indico/modules/events/registration/templates/display/_regform_info.html

```diff
@@ -7,17 +7,29 @@
                 <i class="icon icon-time"></i>
                 <div class="text">
                     <div>
-                        <span class="label">
-                            {% trans %}From{% endtrans %}
-                        </span>
-                        <span class="datetime">
-                            {{ regform.start_dt|format_date(timezone=regform.event.tzinfo) }}
-                        </span>
+                        {% if regform.has_started %}
+                            <span class="label">
+                                {% trans %}Registration is open{% endtrans %}
+                            </span>
+                        {% else %}
+                            <span class="label">
+                                {% trans %}Opens{% endtrans %}
+                            </span>
+                            <span class="datetime">
+                                {{ regform.start_dt|format_date(timezone=regform.event.tzinfo) }}
+                            </span>
+                        {% endif %}
                     </div>
                     <div>
-                        <span class="label">
-                            {% trans %}To{% endtrans %}
-                        </span>
+                        {% if regform.has_started %}
```

Comment:
What about having the if/else inside the `span` block so you avoid repetition?

Reply:
Agreed, for this one it makes more sense. Also keep in mind that you can have a registration without an `end_dt`. In this case it'd probably be nicer to not show this block at all if the registration is running - "Registration is open" is kind of obvious.

Reply:
Agreed and agreed. Although, for the second point, I don't like the empty space created by the missing end date span below the "Registration is open" label. Is there a nice way to push it in the middle of the vertical space?

Reply:
That's why I would not show it at all if the registration is open and has no end date

Reply:
I am not sure what you mean. Even if you don't show the span element, the empty space still exists. Check attachment.

![capture du 2018-07-06 11-03-59](https://user-images.githubusercontent.com/5298184/42370185-5bf131dc-810c-11e8-8da6-0360eb1f415a.png)

Reply:
>  In this case it'd probably be nicer to not show this block at all if the registration is running - "Registration is open" is kind of obvious.

I meant hiding that whole block.